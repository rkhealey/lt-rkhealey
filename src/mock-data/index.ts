import profile from './profile.png';

export const user = {
  username: '@yourname',
  imageSrc: profile,
};

export const links = [
  {
    url: 'http://www.google.com',
    text: 'Link One',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Two',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Three',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Four',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Five',
  },
];
