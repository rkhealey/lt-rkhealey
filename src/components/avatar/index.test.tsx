import React from 'react';
import { shallow } from 'enzyme';
import Avatar from '.';

import { user } from '../../mock-data';

describe('Avatar', () => {
  it('Renders', () => {
    const wrapper = shallow(<Avatar {...user} />);
    expect(wrapper).toMatchSnapshot();
  });
});
