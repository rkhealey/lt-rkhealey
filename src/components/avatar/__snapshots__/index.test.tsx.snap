// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Renders 1`] = `
<Styled(styled.div)
  alignItems="center"
  flexDirection="column"
  mb={5}
>
  <ForwardRef
    alt="@yourname"
    mb={3}
    src="profile.png"
    variant="avatar"
  />
  <ForwardRef>
    @yourname
  </ForwardRef>
</Styled(styled.div)>
`;
